Source/DataTypes/SvgUnitCollection.cs

```diff
@@ -13,7 +13,7 @@ namespace Svg
     [TypeConverter(typeof(SvgUnitCollectionConverter))]
     public class SvgUnitCollection : ObservableCollection<SvgUnit>
     {
-        internal bool IsNull { get; set; }
+        public string StringIfEmpty { get; set; }

```

Comment:
Should this be public? Wouldn't `internal` be better? Also I'm not sure about the name, it may be a bit misleading, though I'm struggling to find a better one, too. Maybe `StringForEmptyValue` or something like this?

Other that that, it all looks good - thanks!

Reply:
Thank you for your review.

> Should this be public?

Make `public` because it is necessary when generating SVG from user code.

> Also I'm not sure about the name, it may be a bit misleading, though I'm struggling to find a better one, too. Maybe StringForEmptyValue or something like this?

I use in sense that collection is empty.(like IsEmpty)

Reply:
Ah, ok - naming is hard, let's go with that then. Thanks!